README.md

```diff
@@ -27,7 +27,7 @@ Because of possible binary incompatibilities here are the dependency versions us
 |         0.1        |     3.1.1     |     0.5.0    |
 |         0.2        |     3.1.1     |     0.6.0    |
 |         0.3        |     3.1.1     |     0.7.0    |
-
```

Comment:
A bit of weirdness. This `README` is not to be edited manually but is rather generated using `tut` from `src/main/tut/README.md`. Edits should be made there, and the `tut` should be run. That way we ensure that the main `README.md` examples are compilable. 

The idea was to use `sbt-release` and have `tut` generation as a release task but I never got around to it. 

Please suggest any improvements here if you know of a better way to ensure the readme examples work via tut that's less confusing!